trino-connector/integration-test/trino-test-tools/trino-cascading-env/init/trino-local/init.sh

```diff
@@ -0,0 +1,84 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+#
+
+set -ex
+trino_conf_dir="$(dirname "${BASH_SOURCE-$0}")"
+trino_conf_dir="$(cd "${trino_conf_dir}">/dev/null; pwd)"
+
+cp "$trino_conf_dir/config/config.properties" /etc/trino/config.properties
+cp "$trino_conf_dir/config/jvm.config" /etc/trino/jvm.config
+cp "$trino_conf_dir/config/log4j2.properties" /etc/trino/log4j2.properties
+cp "$trino_conf_dir/config/catalog/gravitino.properties" /etc/trino/catalog/gravitino.properties
+cp "$trino_conf_dir/config/catalog/trino.properties" /etc/trino/catalog/trino.properties
+
+#start the gravitino server
+cp -r /opt/gravitino-server /tmp/gravitino-server
+rm -fr /tmp/gravitino-server/logs
+rm -fr /tmp/gravitino-server/data
+/tmp/gravitino-server/bin/gravitino.sh start
```

Comment:
When will you stop the Gravitino server? I haven't noticed such logic in the PR.

Reply:
The Gravitino server was stopped by the container shutdown.